examples/all/i2c_example.cc

```diff
@@ -22,7 +22,8 @@ static void read_temperature_sensor_value(Mmio<OpenTitanI2c> i2c,
 	i2c->blocking_write(0x48, buf, 1, false);
 	if (i2c->blocking_read(0x48, buf, 2u))
 	{
-		uint16_t temp = (buf[0] << 8) | buf[1];
+		int16_t regValue = (buf[0] << 8) | buf[1];
+		int64_t temp     = regValue * 1000000LL / 128;
 		Debug::log("The {} readout is {}", regName, temp);
```

Comment:
Units would be great to have in the log.

```suggestion
		Debug::log("The {} readout is {} microdegrees Celcius", regName, temp);
```

*Also, am I right in thinking the temperature is microdegrees not millidegrees after the conversion?*